Bring the changes to current version (#1080)
genesiscommunitysuccess/docs: versioned_docs/version-2023.1/01_getting-started/03_go-to-the-next-level/06_forms.md

  :jsonSchema=${(x) => x.jsonSchema}
  :data=${(x) => x.editedEntity}
></foundation-form>
```

Example schema:
```typescript title='form.ts'
const uiSchema: UiSchema = {
  type: 'VerticalLayout',
  elements: [
    {
      type: 'Control',
      label: 'Address',
      scope: '#/properties/COMPANY_ADDRESS',
    },
    {
      type: 'Control',
      label: 'Issuer Name',
      scope: '#/properties/ISSUER_NAME',
    },
  ],
};

const jsonSchema: JSONSchema7 = {
  type: 'object',
  properties: {
    COMPANY_ADDRESS: {
      oneOf: [
        {
          type: 'null',
        },
        {
          type: 'string',
        },
      ],
      description: 'kotlin.String',
    },
    ISSUER_NAME: {
      type: 'string',
      minLength: 3,
      description: 'kotlin.String',
    },
    ...
  },
  additionalProperties: false,
  required: ['ISSUER_NAME'],
};
```

The `jsonSchema` can either be defined locally (as above) or be fetched from the server based on a resource name.

```jsx
const jsonSchemaResponse = await this.connect.getJSONSchema(this.resourceName);
this.jsonSchema = jsonSchemaResponse?.INBOUND?.properties?.DETAILS as JSONSchema7;
```

### Custom validation

> When defining your `jsonSchema`, you can pass a custom validation function to any field. it is passed as `validateFn` in the `options` block.

Reviewer: @wjhendry 
```suggestion
When defining your `jsonSchema`, you can pass a custom validation function to any field via the  `validateFn` key n the `options` block.
```
